src/components/Main/Main.js

```diff
@@ -0,0 +1,49 @@
+import './Main.css';
+import CountryCard from '../CountryCard/CountryCard';
+import background from '../../un.png';
+import { useCountries } from '../../hooks/useCountries';
+import { useState } from 'react';
+
+export default function Main() {
+  const [order, setOrder] = useState('asc');
+  const [searchName, setSearchName] = useState('');
+  const [loading, setLoading] = useState(false);
+  const { countries, error } = useCountries(order, searchName, setLoading);
+  const [continent, setContinent] = useState('all');
+  //Set removes duplicates
+  const continents = [...new Set(countries.map(({ continent }) => continent))];
+  // const names = [...new Set(countries.map(({ name }) => name))];
+
+  const filtered = countries.filter(
+    (country) => country.continent === continent || continent === 'all'
+  );
+
+  // const nameFilter = countries.filter((country) => country.name === name || name === 'all');
+
+  return (
+    <main style={{ backgroundImage: `url(${background})` }}>
+      <div className="select-container">
+        <p>{loading ? `loading...` : `loaded`}</p>
```

Comment:
You probably don't want loaded to show up on your page I can show you how to make this ternary include everything else you want to display instead of loaded.